mindjunkies/payments/models.py

```
@@ -43,18 +43,14 @@ class Meta:
         unique_together = ("user", "course")
 
     def __str__(self):
-        return self.user.username + " - " + self.course.title
-    
-
+        return self.user.username + " bought - " + self.course.title 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)